system/Security/Security.php

```diff
@@ -125,7 +125,9 @@ class Security implements SecurityInterface
      */
     public function __construct(App $config)
     {
-        /** @var SecurityConfig */
+        /**
+         * @var SecurityConfig $security
+         */
```

Comment:
How come this was forced to multi-line when the same thing in **Events.php** reverted to a single line?

Reply:
this change is my manual change as it remove `@var` due to no variable definition after `@var` type. Updated to single line.